tests/drivers/spi/spi_slave/testcase.yaml

```diff
@@ -5,6 +5,8 @@ tests:
     harness: ztest
     harness_config:
       fixture: gpio_spi_loopback
-    platform_allow: nrf52840dk/nrf52840 nrf54l15pdk/nrf54l15/cpuapp
+    platform_allow: |
+      nrf52840dk/nrf52840 nrf54l15pdk/nrf54l15/cpuapp nrf54h20dk/nrf54h20/cpuapp
```

Comment:
Twister will skip this test on nrf54h20 as long as 'spi' is not listed under supported.
See:
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.yaml#L14
and
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpurad.yaml#L14